<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitForm</class>
 <widget class="QWidget" name="InitForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>248</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>187</width>
    <height>248</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>168</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="price4Edit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="price1Edit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="price5Edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Difficulty 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Difficulty 3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Difficulty 4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Difficulty 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="price2Edit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="price3Edit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Difficulty 5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="nextBtn">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>price1Edit</tabstop>
  <tabstop>price2Edit</tabstop>
  <tabstop>price3Edit</tabstop>
  <tabstop>price4Edit</tabstop>
  <tabstop>price5Edit</tabstop>
  <tabstop>cancelBtn</tabstop>
  <tabstop>nextBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
